### ISSUE TITLE
Add an option to control line spacing

### ISSUE THREAD

**Issue by yanirelfasy** [norbit8/MoodleBooster#65]
# What
Add a slider to the `Enhance Page` section that will allow to control the  line spacing in the page.

# Why
Many accessibility menus allow to control the line spacing. this allow people to control how much space should be between lines 
at the page. 
This will allow people to distinguish text lines and see the page in a way they find more comfortable.

**Comment by yanirelfasy** [comment]
Fixed at #110
